### ISSUE TITLE
Tesla access token security issue

### ISSUE THREAD

**Issue by teslaman2** [Kemmey/Kemmey-TeslaWatch-Public#34]
On the Tesla Access Token website https://shareshortcuts.com/shortcuts/256-tesla-access-token.html I see the following warning:
“Be aware: This shortcut has 1 URL-calls that potentially can compromise your security“. Do I need to be concerned about the security of my credentials when it is sent to Tesla through this API to receive the token?

**Comment by Kemmey** [comment]
Hey @teslaman2 

If you tap the Show button on that warning, you'll see that the external url they're warning about is this url: 

https://owner-api.teslamotors.com/oauth/token

And that url is the OFFICIAL url of the Tesla API. This is NOT a 3rd party api or service - this is DIRECTLY hosted by Tesla. And this is the ONLY place you should ever send your login credentials. 

If you choose to input your username and password into my app, I will send those to that exact url ONCE and then discard your credentials in favor of storing the returned token. 

If you prefer to not input your username and password into my app - which you should, just common sense - you can obtain that token yourself through a lot of different scripts and tools. I prefer shortcuts, because the shortcuts app is made by apple, and I trust apple to not mess around with network traffic. And the shortcuts code is short and easy to understand and particularly, its easy to see what network traffic it will generate. Which translates into trust - I trust that I can input my username and password into something that I can have full transparancy of the execution. 

Long story short - always be wary of entering your username and password. Always input a token - or a refresh token, that will last forever, a normal token will only last 48 days - and generate token with a trusted script or utility.

**Comment by teslaman2** [comment]
Thank you so much for your quick and thorough response!
